extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1816,5 +1816,49 @@ describes.realWin(
         toggleExperiment(win, 'amp-story-branching', false);
       });
     });
+
+    describe('amp-story playable', () => {
+      it('should set playable to true if page has autoadvance', async () => {
+        await createStoryWithPages(1, ['cover'], true);
+
+        await story.layoutCallback();
+        story
+          .layoutCallback()
+          .then(() =>
```

Comment:
We prefer `await` over `.then()`, could you change this and below to

```
await story.layoutCallback()
...
```

Reply:
+1, let's not use any `.then()` in our unit tests :))